pkg/descriptors/controls.go

```diff
@@ -115,6 +124,26 @@ func (vpcc *VideoProbeCommitControl) UnmarshalBinary(buf []byte) error {
 	return nil
 }
 
+// Control Request for Auto-Exposure Mode as defined in UVC spec 1.5, 4.2.2.1.2
+type AutoExposureModeControl struct {
+	Mode AutoExposureMode
+}
+
+func (fac *AutoExposureModeControl) MarshalSize() int {
```

Comment:
```suggestion
func (aemc *AutoExposureModeControl) MarshalSize() int {
```

Use a more descriptive variable name here.